app/models/Prepaid.coffee

```diff
@@ -43,6 +44,16 @@ module.exports = class Prepaid extends CocoModel
       
     return 'available'
 
+  courseType: ->
+    type = @get('type')
+    if type == 'starter_license'
+      return 'starter license'
+    includedCourseIDs = @get('includedCourseIDs')
+    if includedCourseIDs
+      return 'customized license with '+ (includedCourseIDs.map (id) -> utils.courseAcronyms[id]).join('+')
+    else
+      return 'full license'
```

Comment:
We should capitalize these (`'Starter License'` instead of `'starter license'`, etc.), maybe internationalize them.

Reply:
okay, i capitalized them now. but in fact these strings are just use as keys of an invisible json. so i think it is not really important haha. :)